nixos/modules/services/networking/rpki-client.nix

```
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.rpki-client;
+  tals = [
+    "${cfg.package}/etc/rpki/afrinic.tal" "${cfg.package}/etc/rpki/apnic.tal"
+    "${cfg.package}/etc/rpki/lacnic.tal" "${cfg.package}/etc/rpki/ripe.tal" ]
+    ++ optional cfg.accept-arin-rpa "${pkgs.rpki-arin-tal}/arin.tal"
+    ++ cfg.additional-tals;
+in {
+  options = {
+    services.rpki-client = {
+      enable = mkEnableOption (lib.mdDoc "Enable OpenBSD rpki-client");
+
+      package = mkOption {
+        description = (lib.mdDoc "OpenBSD rpki-client");
```

```suggestion
        description = lib.mdDoc "OpenBSD rpki-client";
```
We don't need these. Please remove them in similar places, too.